src/graphql/parseAssessmentRiskLogic.ts

```diff
@@ -4,8 +4,10 @@ import * as t from 'io-ts';
 
 export const AssessmentRiskLogic = t.intersection([
   t.partial({
-    riskAssignment: t.type({
+    riskAssignment: t.partial({
```

Comment:
Perhaps we can define a RiskAssignmentCodec as a schemaToCodec of [RiskAssignmentInput](https://github.com/transcend-io/main/blob/ba8af6bc3d03349003a8a171ea3bab94892de10f/frontend-support/datamap-types/src/schema/assessmentQuestion.ts#L117)?

Reply:
It would be ideal to define these in privacy-types, but I don't think we can import the RiskAssignmentInput schema into it 🤷

Reply:
i dont think this has a graphql codec